### ISSUE TITLE
InstantiationService has been disposed

### ISSUE THREAD

**Issue by almonadavid** [posit-dev/positron#13420]
#### Positron and OS details:

Positron Version: 2026.05.0 (user setup) build 179
Code - OSS Version: 1.109.2
Commit: b271b10dfb5190a18813b46705216dd95e7e8927
Date: 2026-04-30T08:35:02.744Z
Electron: 39.3.0
Chromium: 142.0.7444.265
Node.js: 22.21.1
V8: 14.2.231.22-electron.0
OS: Windows_NT x64 10.0.26200

#### Session details:

Python 3.12.6

## Describe the issue:

A warning or error message, "InstantiationService has been disposed," appears at the bottom-left of the screen. Cannot delete typed code or text in cell chunk, among other things.
The problem occurs in a notebook using both R and Python kernels.
This problem has also occurred in different project folders.

## Steps to reproduce the issue:

1. Jupyter notebook (R or Python Kernel)
2. Use the notebook as normal, but after a while, the issue begins to present itself (See screen recording attached).

## Expected or desired behavior:

Should be able to edit Notebook freely.

## Were there any error messages in the UI, Output panel, or Developer Tools console?

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/f4ad7fd5-6b61-455c-8229-65527d9f780b" />

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/edf5bd03-ca75-4753-b3a7-add461650d6b" />

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/9f850289-b991-4c43-a291-72bcd3c83eaa" />

## Screen recording

https://github.com/user-attachments/assets/3234d5f8-5d1b-4f7a-b769-2a5b8a0fab37

**Comment by rodrigosf672** [comment]
@almonadavid, thank you so much for reporting this issue. I faced this issue on Monday as well, and I'm glad you have documented it so well.

### Notes for Triage

> Try to delete or move arrows in the code. Notice the error message and user cannot backspace/arrow into the code cell. Error is `InstantiationService has been disposed`. Workaround is inconvenient (i.e., using mouse pointer and `Cmd+X`).

As for exact repro steps to consistently repro the bug, as the user reported, it can occur in multiple files and project folders, with no specific triggering steps identified. cc @cindyytong @seeM

**Comment by MathewBlanc** [comment]
I have updated to the daily build and I am still getting this issue. I haven't been able to pin down exact causes. I have tried only having one document open at a time, but it's still happening. I just had it happen when I tried to use the keyboard shortcut to comment out a block from some ggplot code. I tried creating a simple repro example doing this but it worked fine. 

Any guidance on how to pin it down would be appreciated. At this rate it feels like it's happening super regularly (it's happened multiple times today alone, after updating), but it has only been an issue in maybe the last week?


Version: Positron Version: 2026.06.0 build 54
Code - OSS Version: 1.111.0
Commit: 28182bd048d7e62291661384d1fde3f3d59c5ee3
Date: 2026-05-07T17:44:48-07:00
Commit: 39.6.0
Date: 142.0.7444.265
Electron: 22.22.0
ElectronBuildId: 14.2.231.22-electron.0
Chromium: Darwin arm64 24.6.0
Node.js: undefined
V8: undefined
OS: Mac Sequoia 15.6.1

**Comment by juliasilge** [comment]
@MathewBlanc Thank you for your patience! We ran into some bumps with our build process over the weekend, but we have a new [daily build](https://positron.posit.co/updating.html#daily-builds) ready to go now that has this fix. If you can try to update one more time and get 2026.06.0 build 64, you should see the correct behavior.

**Comment by MathewBlanc** [comment]
@juliasilge Great, thanks, I'll give it a go!

**Comment by ToddWatkinsDev** [comment]
This issue seems to have been fixed on the latest update as of 16/05/2026 000:57 UTC

**Comment by AyomideA-S** [comment]
> This issue seems to have been fixed on the latest update as of 16/05/2026 000:57 UTC

Yes, I'm no longer facing this issue after updating to the last daily update.
